index.html

```diff
@@ -181,6 +181,26 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+  var fireballSize = 22;
+  var getFireballSpeed = function(left) {
+    if (left) {
```

Comment:
здесь лучше воспользоваться тернарным оператором
```suggestion
    return left ? 5 : 2;
```